command.go

```diff
@@ -1717,6 +1717,9 @@ func (cmd *baseCommand) execute(ifc command, isRead bool) error {
 		// set command node, so when you return a record it has the node
 		cmd.node, err = ifc.getNode(ifc)
 		if cmd.node == nil || !cmd.node.IsActive() || err != nil {
+			if err == nil {
+				err = NewAerospikeError(INVALID_NODE_ERROR, "Node not found for partition or not active")
```

Comment:
very very minor - but maybe better `"Node not found/not active for partition"?

Reply:
Sure, fixed.